sky/jobs/state.py

```diff
@@ -110,6 +111,12 @@ def create_table(cursor, conn):
                                  }))
     db_utils.add_column_to_table(cursor, conn, 'spot', 'local_log_file',
                                  'TEXT DEFAULT NULL')
+    db_utils.add_column_to_table(cursor,
+                                 conn,
+                                 'spot',
+                                 'metadata',
+                                 'TEXT DEFAULT "{}"',
```

Comment:
```suggestion
                                 'TEXT DEFAULT \'{}\'',
```
String literals in SQL should use single quotes. See #4806.